helper/resource/teststep_providers.go

```diff
@@ -5,10 +5,15 @@ package resource
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
 	"strings"
+
+	"github.com/hashicorp/go-version"
 )
 
+const tfBlockMinReqTFVersion = "1.6.0"
```

Comment:
I think we discussed this out of band, but it might be good to set this to 1.0.0 with the reasoning that we'd eventually remove this version check entirely in the terraform-plugin-testing major version we drop protocol version 5 support. Another potentially good reason to do this now is that it won't potentially delay feedback for folks who may be already be testing on 1.x and upgrade their terraform-plugin-testing.

If you could also drop a quick code comment here about why this version check exists at all, that would be fantastic!